plugins/net_plugin/net_plugin.cpp

```diff
@@ -2237,6 +2237,10 @@ namespace eosio {
             return true;
          }
 
+         if(!cp->is_participating()) {
```

Comment:
Sorry, this block should go before line 2231 to avoid bothering to create the send_buffer

Reply:
moved the test prior to creating send_buffer